### ISSUE TITLE
Add a bv_python command passing all parameters to bv python

### ISSUE THREAD

**Issue by sapetnioc** [populse/soma-base#24]
Some programs needs to launch python with complex parameters including `--`. But this parameter seems to be removed by `bv` command. For instance, I tried to create a `python` command in my build tree (just a link to `bv` as other commands) and use this as Python backed in VS Code. It works to run scripts (they are launched in the casa_distro container). But debugging fails because VS Code tries to launch the following command:
```
/usr/bin/env /home/yann/dev/main/bin/python /home/yann/.vscode/extensions/ms-python.python-2022.18.2/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher 44745 -- /home/yann/dev/main/src/capsul/master/capsul/web/__main__.py
```
and fails because: "ValueError: '--' is not in list".

I propose that casa_distro adds a `bv_python` command during stetup. This command would pass all its parameters verbatim to Python in container. Using the name `bv_python` instead of `python` will prevent to hide system python when `bin` directory is in `PATH` env. var.

**Comment by sapetnioc** [comment]
For the specific need of debugging with VS Code, `bv_python` command must support to be called from both outside and inside the container. I successfully started a step by step debug with the following content for `bv_python`:
```sh
#!/bin/sh
if [ -z ${CASA_CONF+x} ]; then
    # Here, we are outside the casa-distro container
    # => call python through bv command
    exec "$(dirname $0)/bv" python -- "$@"
else
    # Here, we are in the casa-distro container
    # => simply call python
    exec python "$@"
fi
```

**Comment by denisri** [comment]
OK but this situation is not really specific to python: virtually any command run inside the container through `bv` will run into the same problem if `--` is in its parameters, right ?
Maybe the best thing to do would be to modify `bv`/`casa_distro` arguments parsing to handle the `--` options (`bv` and `casa_distro run` would pass those arguments to the command inside the container, typically)

**Comment by denisri** [comment]
(this issue should be in brainvisa/casa-distro, not in populse/soma-base)

**Comment by sapetnioc** [comment]
Yes, why did I put this issue here ... ? I kept it open and did not realize my mistake.

**Comment by denisri** [comment]
(it doesn't seem possible to transfer an issue to a different organization/owner in github...)

**Comment by sapetnioc** [comment]
I think `--` was introduced in `bv` to allow to pass options to a command that could be considered as `bv` options (for instance `-h` and `-v`). But you are right, all commands that are defined as symlinks to `bv`must send all their parameters to the target command in the container. We can probably easily do that by inserting `--` in `sys.argv[1]` when such a symlink is detected.

**Comment by sapetnioc** [comment]
I will try to transfer the issue by copying it with the API 🤞

**Comment by denisri** [comment]
It's maybe not worth losing time on the ticker transfer. You may just open another one in casa-distro and link it to this one.

**Comment by sapetnioc** [comment]
This issue had been fixed by brainvisa/casa-distro#315
